ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -109,7 +109,7 @@
                 <name-part name="family" font-variant="small-caps"/>
               </name>
               <et-al font-style="italic"/>
-              <label form="short" prefix="&#160;(" suffix=".)"/>
+              <label form="short" prefix=" (" suffix=".)"/>
```

Comment:
Hi rmzelle, thanks for your comments. My intention is to have normal spaces here, as I see no justification for having non-breaking spaces instead.